src/localconf.js

```diff
@@ -101,6 +104,22 @@ function buildConfig(opts) {
 
   let useGuppyForExplorer = false;
 
+  let studyViewerConfig = {
+    openMode: 'open-all',
+    defaultOpenStudyName: '',
+    accessibleValidationField: undefined,
+    dataType: undefined,
+  };
+  if (config.studyViewerConfig) {
+    useGuppyForExplorer = true;
```

Comment:
because if not then portal will try to talk to arranger 🤪 

will be fixed once we start https://ctds-planx.atlassian.net/browse/PXP-6409, which is pending because we want to be sure that VA is stablized with Guppy